cfg.json

```diff
@@ -1,11 +1,13 @@
 {
-    "Version": "v0.5.3",
+    "Version": "v0.5.4",
     "SourceDir": "",
     "OutputDir": "",
     "MeleeSplash": true,
     "IncreaseStackSizes": true,
     "IncreaseMonsterDensity": 1,
     "EnableTownSkills": true,
+    "BiggerGoldPiles": true,
```

Comment:
should all these options be false by default?

Reply:
Definitely.

Reply:
Options updated to all off except startswithcube